lib_test/test_data/storage/requests/DATA.MIRROR.list.request.0

```diff
@@ -0,0 +1 @@
+<?xml version="1.0"?><methodCall><methodName>DATA.MIRROR.list</methodName><params><param><value><struct><member><name>dbg</name><value>dbg</value></member></struct></value></param></params></methodCall>
```

Comment:
Nice to have all these tests, how were they generated? Did you use the old camlp4 code to generate it?

Reply:
Ah no, I'm afraid I didn't - it's _way_ easier to generate the test data using the new style IDL.